README.md

```diff
@@ -70,7 +70,7 @@ To ensure the long-term quality of the GX Core codebase, we're not yet ready to
 | -------------------- | ------------------ | ----- |
 | CredentialStore      | 🟢 Ready           |       |
 | BatchDefinition      | 🟡 Partially ready | Formerly known as splitters |
-| Action               | 🔴 Not ready       |       |
+| Action               | 🟢 Ready           |       |
```

Comment:
**Note for product reviewer** Please confirm the contribution readiness status for Actions. I tried to ask about this in my docs plan but didn't get an answer so want to double check that I've got the right status.

Reply:
Not the product review but LGTM from an engineering perspective - the API is stable and the steps to create custom actions are quite simple (as laid out here 🎉 )